### ISSUE TITLE
关于有标签数据与未标记数据之间相似度矩阵的计算问题

### ISSUE THREAD

**Issue by InMyDreammer** [xmed-lab/AllSpark#19]
您好，我想请问下您在model/semseg/allspark.py文件中的第82行中“cls_sim_matrix = torch.matmul(F.normalize(pseudo_prob_map, dim=-1), F.normalize(emb_u, dim=1))”，为什么在计算相似度矩阵时，pseudo_prob_map与emb_u均来自于未标记数据？
